src/components/Dashboard/FilingHistoryList/LimitedRestorationExtensionFiling.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+  <div v-if="filing" class="limited-restoration-extension-filing body-2">
+    <h4>Extension of Limited Restoration</h4>
+
+    <p>
+      The period of restoration was successfuly extended and is active
+      <strong>until {{ expiryDateFriendly }}</strong>. At the end of the extended limited
+      restoration period, the company will be automatically dissolved. If you require assistance
+      to extend a limited restoration/reinstatement or wish to convert your restoration from a
+      limited period to a full restoration, please contact BC Registries staff:
+    </p>
+
+    <ContactInfo class="mt-4" />
+  </div>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import { Component, Prop } from 'vue-property-decorator'
+import { ContactInfo } from '@/components/common'
+import { HistoryItemIF } from '@/interfaces'
+import { DateUtilities } from '@/services'
+
+@Component({
+  components: { ContactInfo }
+})
+export default class LimitedRestorationExtensionFiling extends Vue {
+  /** The subject filing. */
```

Comment:
Since this is a new component, what do you think about using the more the Options API rather than the Class API so it's natively supported by Vue3.

Reply:
A quick Google search suggests Class API is well supported in Vue3:
https://github.com/vuejs/vue-class-component/issues/402
https://levelup.gitconnected.com/from-vue-class-component-to-composition-api-ef3c3dd5fdda
https://medium.com/@takeit_/vue-3-composition-api-with-typescript-class-components-dbe7cc9bb468

In THIS PR, I am following existing code. In my next (refactoring) PR, I will look at this and change it over if it's not too much of a PITA, ok?

Reply:
Apologies, that's my mistake, when we discussed this issue previously (see below) I shared a link that I misread to read the Class API support had been abandoned in Vue3.  

![image](https://user-images.githubusercontent.com/25233684/226975484-e894d696-ca0a-493a-b8df-d66f29bd8ce8.png)

Reply:
Are you OK if I don't change this component to Options API, and if so can you please resolve this conversation? TIA